docs/src/docs/app.tsx

```diff
@@ -331,6 +338,18 @@ const DocsApp = () => (
       </SimpleGrid>
     </Fade>
 
+    <Card w="70%" mx="auto" bgColor="bg.layer" borderRadius="md" maxH="4xl" overflowY="scroll">
```

Comment:
`<Card w="70%" mx="auto" bgColor="bg.layer" borderRadius="md" maxH="4xl" overflowY="scroll">`

Please put the stylings in a new row for better code readability. 

```
<Card 
w="70%" 
mx="auto" 
bgColor="bg.layer"
borderRadius="md" 
maxH="4xl" 
overflowY="scroll"
>
```